Parse-Dashboard/parse-dashboard-config.json

```diff
@@ -1,14 +1,133 @@
 {
   "apps": [
     {
-      "serverURL": "http://localhost:1338/parse",
-      "appId": "hello",
-      "masterKey": "world",
-      "appName": "",
-      "iconName": "",
-      "primaryBackgroundColor": "",
-      "secondaryBackgroundColor": ""
+      "serverURL": "https://parseapi-homolog.back4app.com",
```

Comment:
Please remove this file and I recommend to invalidate all exposed keys and secure your apps.

Reply:
Thanks @mtrezza